.update/do.py

```diff
@@ -0,0 +1,131 @@
+import click
+import requests
+from pathlib import Path
+
+from cnxcommon import ident_hash
+
+
+here = Path(__file__).parent
+CNX_HOST = 'archive.cnx.org'
+MAP_FILEPATH = here.resolve().parent / 'roles/webview/files/etc/nginx/uri-maps/rex-uris.map'
+
+
+def get_rex_release_json_url(host):
+    return f'https://{host}/rex/release.json'
+
+
+def get_book_slug(book_id):
+    url = (
+        "https://openstax.org/apps/cms/api/v2/pages/"
```

Comment:
Very minor thing, should not it be `f"https://openstax.org/apps/cms/api/v2/pages/"` too?

Reply:
It doesn't actually have any formatting in it. It'd just be running the string formatting operation over it for no reason. But I could change it if you'd like.

Reply:
It's really a non issue.